### PR TITLE
Fix small link issue

### PR DESCRIPTION
**Pull request by wrt54g**
Fix issue related to #2919

### CODE REVIEW

File: configurator/DOCS.md [hunk 1]
```diff
@@ -71,6 +71,6 @@ In case you've found a bug, please [open an issue on our GitHub][issue].
 
 [discord]: https://discord.gg/c5DvZ4e
 [forum]: https://community.home-assistant.io
-[issue]: https://github.com/home-assistant/hassio-addons/issues
+[issue]: https://github.com/home-assistant/addons/issues
 [reddit]: https://reddit.com/r/homeassistant
-[repository]: https://github.com/addons/repository
+[repository]: https://github.com/hassio-addons/repository
```
Comment (agners) [inline configurator/DOCS.md:76]: That seems wrong

Comment (wrt54g) [reply configurator/DOCS.md:76]: @agners Why?

Comment (wrt54g) [reply configurator/DOCS.md:76]: This fixes an issue with the larger PR #2919.

Comment (agners) [reply configurator/DOCS.md:76]: Hm I see. But that URL is not really used here, we don't point to the community add-ons.

Just drop it then.

```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks! 👍 

../Frenck

**Pull request merged**